src/ToricVarieties/NormalToricVarieties/methods.jl

```diff
@@ -3,21 +3,21 @@
 ############################
 
 @doc Markdown.doc"""
-    betti_number(v::AbstractNormalToricVariety, i::Int)
+    betti_numbers(v::AbstractNormalToricVariety, i::Int)
 
 Compute the i-th Betti number of the normal toric variety `v`.
 """
-function betti_number(v::AbstractNormalToricVariety, i::Int)
+function betti_numbers(v::AbstractNormalToricVariety, i::Int)
     # check input
     if i > 2*dim(v) || i < 0
         return 0
     end
     
     # extract vector of currently-known Betti numbers (or create it if necessary)
-    if !has_attribute(v, :betti_number)
+    if !has_attribute(v, :betti_numbers)
         betti_numbers = fill(fmpz(-1),2*dim(v)+1)
     else
-        betti_numbers = get_attribute(v, :betti_number)::Vector{fmpz}
+        betti_numbers = get_attribute(v, :betti_numbers)::Vector{fmpz}
```

Comment:
By the way, the above few lines could be rewritten using `get_attribute!`, too. Instead of adding a _betti_nimbers helper